ibm/config.go

```diff
@@ -872,8 +884,9 @@ func (c *Config) ClientSession() (interface{}, error) {
 	}
 
 	// IBM Network CIS DNS Record service
+	cisEndPoint := envFallBack([]string{"IBMCLOUD_NETWORK_CIS_ENDPOINT"}, "https://api.cis.cloud.ibm.com")
```

Comment:
please use `IBMCLOUD_CIS_API_ENDPOINT` instead of `IBMCLOUD_NETWORK_CIS_ENDPOINT`

Reply:
done